collectors/applications.go

```diff
@@ -242,6 +258,18 @@ func (c ApplicationsCollector) reportApp(application models.Application, objs *m
 		buildpack = appSum.DetectedBuildpack
 	}
 
+	// 3. Use the droplet data for the buildpack metric
+	apps, exists := objs.Apps[application.GUID]
```

Comment:
is there a specific reason to get the application from the obj.Apps slice here ?

Reply:
I had the droplet logic in mind when initially reviewing the code, which is why I included that line. However, considering the current implementation, you're right. The line of code fetching the application from `objs.Apps` is unnecessary and can be removed. Thank you for pointing that out.